packages/adapter-prisma/lib/adapter-prisma.js

```diff
@@ -0,0 +1,611 @@
+          };
+        }
+        return this.fieldAdaptersByPath[path] && this.fieldAdaptersByPath[path].isRelationship
+          ? value === null
+            ? { disconnect: true }
+            : { connect: { id: Number(value) } }
+          : value;
+      }),
+      include,
+    });
+  }
+
+  async _delete(id) {
+    return this.model.delete({ where: { id: Number(id) } });
+  }
+
+  ////////// Queries //////////
+  async _itemsQuery(args, { meta = false, from = {} } = {}) {
+    const filter = this.prismaFilter({ args, meta, from });
+    if (meta) {
+      let count = await this.model.count(filter);
+      const { first, skip } = args;
+
+      // Adjust the count as appropriate
+      if (skip !== undefined) {
+        count -= skip;
+      }
+      if (first !== undefined) {
+        count = Math.min(count, first);
+      }
+      count = Math.max(0, count); // Don't want to go negative from a skip!
+      return { count };
+    } else {
+      return this.model.findMany(filter);
+    }
+  }
+
+  prismaFilter({ args: { where = {}, first, skip, sortBy, orderBy, search }, meta, from }) {
+    const ret = {};
+    const allWheres = this.processWheres(where);
+
+    if (allWheres) {
+      ret.where = allWheres;
+    }
+
+    if (from.fromId) {
+      if (!ret.where) {
+        ret.where = {};
+      }
+      const a = from.fromList.adapter.fieldAdaptersByPath[from.fromField];
+      if (a.rel.cardinality === 'N:N') {
+        const path = a.rel.right
+          ? a.field === a.rel.right // Two-sided
+            ? a.rel.left.path
+            : a.rel.right.path
+          : `from_${a.rel.left.path}`; // One-sided
+        ret.where[path] = { some: { id: Number(from.fromId) } };
+      } else {
+        ret.where[a.rel.columnName] = { id: Number(from.fromId) };
+      }
+    }
+
+    // TODO: Implement configurable search fields for lists
+    const searchField = this.fieldAdaptersByPath['name'];
+    if (search !== undefined && searchField) {
+      if (searchField.fieldName === 'Text') {
+        // FIXME: Think about regex
+        if (!ret.where) ret.where = { name: search };
+        else ret.where = { AND: [ret.where, { name: search }] };
+        // const f = escapeRegExp;
+        // this._query.andWhere(`${baseTableAlias}.name`, '~*', f(search));
+      } else {
+        // Return no results
+        if (!ret.where) ret.where = { AND: [{ name: null }, { NOT: { name: null } }] };
+        else ret.where = { AND: [ret.where, { name: null }, { NOT: { name: null } }] };
+      }
+    }
+
+    // Add query modifiers as required
+    if (!meta) {
+      if (first !== undefined) {
+        // SELECT ... LIMIT <first>
+        ret.take = first;
+      }
+      if (skip !== undefined) {
+        // SELECT ... OFFSET <skip>
+        ret.skip = skip;
+      }
+      if (orderBy !== undefined) {
+        // SELECT ... ORDER BY <orderField>
+        const [orderField, orderDirection] = orderBy.split('_');
```

Comment:
This would break if you have a field with an _ in it and try to order by it right?

Reply:
Yep, we need to write some tests for this, because I'm pretty certain that this bug exists in the `knex` adapter as well.

Reply:
I'm going to leave this as an open issue until someone can write some tests and we fix this across all adapters.